platform-plugins.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: page
+title: Plugins -- Integrate with platform-specific APIs
+permalink: /platform-plugins/
+---
+
+This guide describes how to connect your Flutter app to platform-specific
+services available on iOS and Android devices. This includes both device APIs
+(like `Clipboard` and `BatteryState`) and third-party platform SDKs (like
+Google Analytics).
+
+* TOC
+{:toc}
+
+## Options for integrating with platform-specific APIs
+
+Mobile devices offer many options to extend your app through platform-specific
+APIs. These are critical to the completeness of most mobile apps. Flutter offers
+the following ways to integrate with these via 'plugins', packages that contains
+an API definition written in Dart, coupled with a platform-specific
+implementation for Android, for iOS, or for both.
+
+There are two ways you can use plugins:
+
+* **Use existing platform plugins:** For frequently used APIs and SDKs, use
+ existing platform plugins, with no need to write any platform-specific code.
+ [See details below](#use).
+
+* **Create a platform plugin:** Write your own platform-specific code using
+ Flutter's Platform Channels. This can be used for private APIs, or for cases
```

Comment:
Here Platform Channel is written in two words, which is probably better, since it is not a Dart type. But we should be consistent.